projects/pt1/python/torch_mlir_e2e_test/configs/torchdynamo.py

```diff
@@ -130,16 +130,17 @@ def my_aot_autograd_backend(gm: torch.fx.GraphModule,
 
     my_backend = aot_autograd(fw_compiler=my_aot_autograd_backend,
                               decompositions=_get_decomposition_table)
-
     with torch.no_grad():
         set_model_name(model.__class__.__name__)
         torch._dynamo.reset()
         dynamo_f = dynamo.optimize(my_backend, nopython=True)(
             lambda method, *inputs: method(*inputs))
         dynamo_f(lambda *inputs: model(*[x.clone() for x in inputs]),
                  *example_args)
+        print(mlir_module)
```

Comment:
1. possible debug statement left in
2. this is duplicated around something that apparently does not change the mlir module

Reply:
Debug statement that leaked through. I'll remove.